packages/edit-site/src/components/sidebar-edit-mode/index.js

```diff
@@ -51,7 +51,7 @@ export function SidebarComplementaryAreaFills() {
 			supportsGlobalStyles: ! settings?.supportsTemplatePartsMode,
 			isEditingPage:
 				select( editSiteStore ).isPage() &&
-				select( editorStore ).getRenderingMode() !== 'template-only',
+				select( editorStore ).getCurrentPostType() !== 'wp_template',
```

Comment:
What's the impact of this boolean here?

Reply:
When editing a page in the site editor and selecting a block it shows the document sidebar tab instead of the block sidebar tab:
![Screenshot 2024-01-16 at 1 47 06 pm](https://github.com/WordPress/gutenberg/assets/677833/c8db4105-c7dd-45cd-8243-d432d0b59e3d)

When editing any other post type the block tab is shown instead.

Using both `isPage` and `getCurrentPostType() !== 'wp_template'` might be a duplication.

Important note - site editor page editing mode seems to be broken in this PR. The page content isn't shown.

Reply:
Yeh, the check for `wp_template` wasn't needed, have removed.